Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAApp/Getting-Started.md

```diff
@@ -62,40 +65,78 @@ Follow the steps [here](https://learn.microsoft.com/virtualization/windowscontai
+```PowerShell 
+az acr login -n $ACRNAME
+```
 
-2. To validate that your cluster is successfully retrieving your GMSA, go into your domain controller local server menu, go to Tools and select Event Viewer. Look under ActiveDirectory events. Look at the contents of the most recent events for a message that says "A caller successfully fetched the password of a group managed service account." The IP address of the caller should match one of your AKS cluster IPs.
+Push the images into the container registry. Ensure you are logged into the Azure Container Registry, you should show a successful login from the command above.
 
-### Deploy workload without support for HTTPS
+```PowerShell
+docker push $ACRNAME.azurecr.io/simpleapp:v1
+```
 
-Navigate to "Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/RatingsApp" folder.
+To verify they have been pushed run the following commands:
 
-1. Update the [manifest file](manifests/deployment_sampleapp.yml) for the sample application with your GMSA name and Windows NodePool(s) name.
-2. Run ``` kubectl apply -f deployment_sampleapp.yml ```
+```PowerShell
+az acr repository show -n $ACRNAME --image simpleapp:v1
+```
+## Deploy workload without support for HTTPS
+
+Navigate to "aks-baseline-windows/Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAAPP/manifests" folder.
+
+1. Update the [manifest file](manifests/deployment_sampleapp.yml) for the sample application with your GMSA name (look for < GMSA Credential Spec Name > in the manifest) and application container image name (Look for < Image Name > in the manifest).
+2. Run ``` kubectl apply -f deployment_sampleapp.yml -n sampleapp ```
 
 ### Check your deployed workload
 
-1. Verify Deployment by executing the following commands:
+1. Verify Deployment by executing the following commands on your jumpbox:
 
 ```Powershell
 kubectl get deployment
-kubectl get pod
+kubectl get pods -n sampleapp
 kubectl describe ingress
 ```
-2. Copy the ip address displayed by running kubectl describe ingress, open a browser, navigate to the IP address obtained above from the ingress controller and explore your website.
-
-3. You can also login to pod by running:
+2. Copy the ip address displayed by running ``` kubectl describe ingress ``` on your jumpbox, open a browser, navigate to the IP address obtained above from the ingress controller and explore your website.
+
+# Next Steps
+- [Troubleshooting](#how-to-validate-your-gmsa-integration-after-deployment)
+- [Try deploying our Legacy .NET Application example](../eshopLegacyApp/Getting-Started.md)
+- [Add support for HTTPS](#deploy-the-ingress-with-https-support)
```

Comment:
Pretty weird branching point here.  The bottom of the page doesn't have any next steps (where do they go when they get to the bottom?)  Maybe you can still clarify where you want the user to go next.  Remember, they eventually need to get to "cleanup," so maybe get them back to the sample app selection page?

Reply:
Moved next steps to bottom and added link to cleanup documentation